src/Compilers/CSharp/Test/Emit3/FieldKeywordTests.cs

```diff
@@ -326,6 +324,198 @@ class C
+                  // Code size        8 (0x8)
+                  .maxstack  2
+                  IL_0000:  ldarg.0
+                  IL_0001:  ldarg.1
+                  IL_0002:  stfld      "object A.<Q3>k__BackingField"
+                  IL_0007:  ret
+                }
+                """);
+            if (!typeKind.StartsWith("record"))
+            {
+                var comp = (CSharpCompilation)verifier.Compilation;
+                var actualMembers = comp.GetMember<NamedTypeSymbol>("A").GetMembers().ToTestDisplayStrings();
+                string readonlyQualifier = typeKind.EndsWith("struct") ? "readonly " : "";
+                var expectedMembers = new[]
+                {
+                    "System.Object A.<P1>k__BackingField",
+                    "System.Object A.P1 { get; set; }",
+                    "System.Object A.P1.get",
+                    "void A.P1.set",
+                    "System.Object A.<P2>k__BackingField",
+                    "System.Object A.P2 { get; set; }",
+                    "System.Object A.P2.get",
+                    "void A.P2.set",
+                    "System.Object A.<P3>k__BackingField",
+                    "System.Object A.P3 { get; set; }",
+                    "System.Object A.P3.get",
+                    "void A.P3.set",
+                    "System.Object A.<Q1>k__BackingField",
+                    "System.Object A.Q1 { get; set; }",
+                    readonlyQualifier + "System.Object A.Q1.get",
+                    "void A.Q1.set",
+                    "System.Object A.<Q2>k__BackingField",
+                    "System.Object A.Q2 { get; set; }",
+                    "System.Object A.Q2.get",
+                    "void A.Q2.set",
+                    "System.Object A.<Q3>k__BackingField",
+                    "System.Object A.Q3 { get; init; }",
+                    "System.Object A.Q3.get",
+                    "void modreq(System.Runtime.CompilerServices.IsExternalInit) A.Q3.init",
+                    "System.Object A.<Q4>k__BackingField",
+                    "System.Object A.Q4 { get; set; }",
+                    readonlyQualifier + "System.Object A.Q4.get",
+                    "void A.Q4.set",
+                    "System.Object A.<Q5>k__BackingField",
+                    "System.Object A.Q5 { get; init; }",
+                    readonlyQualifier + "System.Object A.Q5.get",
+                    "void modreq(System.Runtime.CompilerServices.IsExternalInit) A.Q5.init",
+                    "A..ctor()"
+                };
+                AssertEx.Equal(expectedMembers, actualMembers);
+            }
+        }
+
+        [Fact]
+        public void ImplicitAccessorBody_02()
+        {
+            string source = """
+                interface I
+                {
+                    static object P1 { get; set { _ = field; } }
+                    static object P2 { get { return field; } set; }
+                    object Q1 { get; set { _ = field; } }
```

Comment:
>{ get; set { _ = field; } } [](http://example.com/codeflow?start=30&length=27)

Should this be reported as an error, with `ErrorCode.ERR_InterfacesCantContainFields` perhaps?

Reply:
Will review the tests in depth later, but it feels reasonable for the synthesized field's "location" to be the first usage of the field keyword, and to report the exact error you are saying there.